src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+  health = 100;
```

Comment:
The `health` property should be private to encapsulate it properly. Consider using a private field for this purpose.